crates/release_plz_core/src/next_ver.rs

```diff
@@ -599,6 +599,27 @@ impl Updater<'_> {
         registry_packages: &PackagesCollection,
         repository: &Repo,
     ) -> anyhow::Result<Vec<(&Package, Diff)>> {
+        // Get the source files for each package and save them to a hashmap
+        // This ignores all files in .gitignore ore in the package.exclude field in the Cargo.toml
+        let path = repository.directory();
+        let context = cargo::util::context::GlobalContext::default()?;
+        let ws = cargo::core::Workspace::new(&path.as_std_path().join("Cargo.toml"), &context)?;
```

Comment:
isn't this the same of https://github.com/MarcoIeni/release-plz/blob/986c949b4ecbd5be2fd902fc08a5ae0e4f484df1/crates/release_plz_core/src/package_compare.rs#L105

Reply:
Oh yes. But i get an additional result `Cargo.toml.orig`

Reply:
Should i use your option and just hardcode to skip `Cargo.toml.orig`?

Reply:
Yes please. I hope to remove cargo as a dependency one day https://github.com/MarcoIeni/release-plz/issues/1331

Reply:
already done